packages/rs-dapi-client/src/dapi_client.rs

```diff
@@ -198,6 +217,9 @@ impl DapiRequestExecutor for DapiClient {
         let retries_counter_arc = Arc::new(AtomicUsize::new(0));
         let retries_counter_arc_ref = &retries_counter_arc;
 
+        // We need reference so that the closure is FnMut
+        let applied_settings = &applied_settings;
```

Comment:
applied_settings_ref as above? Shadowing is confusing.

Reply:
done